python/tests/test_pdb.py

```diff
@@ -61,3 +61,64 @@ def test_posterior_database():
 
     datasets = list(pdb.datasets())
     assert len(datasets) > 0
+
+
+def test_posterior_database_github():
+    # Skip test if GITHUB_PAT not defined
+    if "GITHUB_PAT" not in os.environ:
+        return
+
+    pdb = PosteriorDatabaseGithub()
```

Comment:
This test probably should check if it is in CI and then use the correct repo + ref.